Source/WebInspectorUI/UserInterface/Workers/HeapSnapshot/HeapSnapshot.js

```diff
@@ -761,11 +773,26 @@ HeapSnapshot = class HeapSnapshot
             pathsBeingProcessed[i] = undefined;
 
             if (this._nodeOrdinalIsGCRoot[nodeOrdinal]) {
-                let fullPath = currentPath.slice();
                 let nodeIndex = nodeOrdinal * this._nodeFieldCount;
-                fullPath.push({node: nodeIndex});
-                gcRootPaths.push(fullPath);
-                continue;
+
```

Comment:
@dcrousso How can I test that this doesn't break WebInspector?

Reply:
probably just open the Timelines Tab and take a heap snapshot and see if it loads and works correctly.  at the minimum id suggest checking that all the features mentioned in <https://webkit.org/web-inspector/timelines-tab/#javascript-allocations-timeline> and <https://webkit.org/blog/6425/memory-debugging-with-web-inspector/> (especially the part about determining/showing the shortest path) still work correctly